addons/base_import_module/static/src/base_import_list_renderer.js

```diff
@@ -0,0 +1,18 @@
+/** @odoo-module */
+import { ListRenderer } from "@web/views/list/list_renderer";
+
+export class ImportListRenderer extends ListRenderer {
+    get hasSelectors() {
+        return this.props.list.records[0]._values.module_type != 'industries';
```

Comment:
Maybe not 100% relevant here but... Don't we need to ensure that `super` is `true` as well?
```suggestion
        return super.hasSelectors() && this.props.list.records[0]._values.module_type != 'industries';
```